### PR TITLE
Lab2: Add dark mode for lab2 generic dialogs.

### PR DESCRIPTION
**Pull request by cnbrenci**
Updated colors for generic dialogs for lab2 labs to use dark mode colors when theme is dark mode.

Screenshots of the different dialogs after the change:

### aichat (unchanged since it's a light mode lab)
image

### Music Lab:
image

New Variable dialog:
image

### Python Lab:

File Move:
image

New Folder (new file looks the same except the header text)
image

Rename
image

Move File:
image

Delete folder:
image

Delete File:
image

## Links

- jira ticket: [CT-847 [Nice to have] Make a dark mode version of the lab2 modal](https://codedotorg.atlassian.net/browse/CT-847)
- [figma](https://www.figma.com/design/vcNGOFsIks0HhRk6GEOfZf/Upper-Lab?node-id=4648-8064&node-type=rounded_rectangle&m=dev)

## Testing story

Tested locally

## Follow-up work

The [figma](https://www.figma.com/design/vcNGOFsIks0HhRk6GEOfZf/Upper-Lab?node-id=4648-8064&node-type=rounded_rectangle&m=dev) also contains some changes to the layout that I did not include since those will affect all the generic dialogs in both light and dark mode, would probably need a bit more design support for the different types of dialog since we just have one example in figma, and can be done separately, such as:
 - width of the text input
 - `x` in the top right
 - different button layout
 - placeholder text for the different dialogs

[Additional layout updates to this dialog](https://codedotorg.atlassian.net/browse/CT-874)

### CODE REVIEW

File: apps/src/lab2/views/dialogs/GenericPrompt.tsx [hunk 1]
```diff
@@ -46,6 +49,7 @@ const GenericPromptBody: React.FunctionComponent<GenericPromptBodyProps> = ({
         value={prompt}
         onChange={e => handleInputChange(e.target.value)}
         errorMessage={errorMessage}
+        color={theme === Theme.DARK ? 'white' : undefined}
```
Comment (cnbrenci) [inline apps/src/lab2/views/dialogs/GenericPrompt.tsx:65]: buttonColors

**Comment by mikeharv** [comment]
Nice! Would you mind checking a Blockly prompt in Music Lab too, while you're here?

To test this, you can click the Variables category of the toolbox in an Advanced Mode level and then click the "Create variable..." button.

http://localhost-studio.code.org:9000/s/allthethings/lessons/46/levels/5

Thank you!

**Review by molly-moen — APPROVED** [review]

**Comment by cnbrenci** [comment]
Waiting on merging this pending an answer to https://codedotorg.slack.com/archives/C04TH8400AU/p1729206424065559

**Comment by cnbrenci** [comment]
> Nice! Would you mind checking a Blockly prompt in Music Lab too, while you're here?
> 
> To test this, you can click the Variables category of the toolbox in an Advanced Mode level and then click the "Create variable..." button.
> 
> http://localhost-studio.code.org:9000/s/allthethings/lessons/46/levels/5
> 
> Thank you!

Added screenshot to the description!

**Review by thomasoniii — APPROVED** [review]
lgtm 🚀

**Pull request merged**